utils/scripts/ocm/ocm.py

```diff
@@ -90,37 +90,43 @@ def ocm_describe(self, filter=""):
             cmd += " " + filter
         ret = execute_command(cmd)
         if ret is None:
-            log.info("ocm describe for cluster "
-                   "{} failed".format(self.cluster_name))
+            log.info("ocm describe for cluster " "{} failed".format(self.cluster_name))
             return None
         return ret
 
     def is_osd_cluster_exists(self):
         """Checks if cluster exists"""
         ret = self.ocm_describe()
         if ret is None:
-            log.info("ocm cluster with name "
-                   "{} not exists!".format(self.cluster_name))
+            log.info(
+                "ocm cluster with name " "{} not exists!".format(self.cluster_name)
+            )
             return False
         log.info("ocm cluster with name {} exists!".format(self.cluster_name))
         return True
 
     def osd_cluster_create(self):
         """Creates OSD cluster"""
 
-        if ((self.channel_group == "candidate") and (self.testing_platform == "prod")):
-            log.error("Channel group 'candidate' is available only for stage environment.")
+        if (self.channel_group == "candidate") and (self.testing_platform == "prod"):
+            log.error(
+                "Channel group 'candidate' is available only for stage environment."
+            )
             sys.exit(1)
 
         version = ""
         if self.openshift_version != "":
-            version_match = re.match(r'(\d+\.\d+)\-latest',self.openshift_version)
+            version_match = re.match(r"(\d+\.\d+)\-latest", self.openshift_version)
             if version_match is not None:
                 version = version_match.group(1)
                 chan_grp = ""
-                if (self.channel_group == "candidate"):
+                if self.channel_group == "candidate":
                     chan_grp = "--channel-group {}".format(self.channel_group)
-                version_cmd = "ocm list versions {} | grep -w \"".format(chan_grp) + re.escape(version) + "*\""
+                version_cmd = (
```

Comment:
When cutting off lines and going to the next line, shouldn't we use '\' at the end of cut off lines?

Reply:
`black` has auto formatted this piece of code... since it was string concatenation, it prefers to use operators first.

Since it is string concatenation using `+` operator, I don't see a reason why we would need `"` unless we use `format string`